docs/data-builder-dummy-data.md

```diff
@@ -0,0 +1,39 @@
+# Supplying a dummy dataset
+
+---8<-- 'includes/data-builder-danger-header.md'
+
+Because OpenSAFELY doesn't allow direct access to individual patient
+records, researchers must use a **dummy dataset** for developing their analytic
+code on their own computer.
```

Comment:
And when pushing to GitHub, no? How about...

```suggestion
code.
```

or alternatively, using the terminology from the [Security](https://docs.opensafely.org/security-levels/) section...

```suggestion
code outside the EHR vendor's secure environment on the OpenSAFELY server.
```

Reply:
Ah yes, I copied this verbatim from the existing docs, but your suggestions are improvements.